### ISSUE TITLE
problem: auto repeat patch breaks revanced 

### ISSUE THREAD

**Issue by Botan-Yuukan** [revanced/revanced-patches#146]
# 🐞 Issue

Revanced crashes if you click on any video due to the autorepeat-by-default patch making revanced complacently unusable.

# ⚙ Reproduce

Include autorepeat-by-default patch in build

# 🛠 Solution

Remove the autorepeat-by-default patch

# ⚠ Additional context

none

**Comment by bogadana** [comment]
youtube version?

**Comment by Botan-Yuukan** [comment]
17.26.35

**Comment by Sculas** [comment]
I suppose this is because resources aren't merged?

**Comment by TheJeterLP** [comment]
> I suppose this is because resources aren't merged?

No it doesn't need resources. I'll have a look at it tomorrow, I wrote that patch.

**Comment by bogadana** [comment]
I tested 17.26 yesterday and it worked. Unless this is excluded by default?

**Comment by TheJeterLP** [comment]
> I tested 17.26 yesterday and it worked. Unless this is excluded by default?

It is excluded yes

**Comment by bogadana** [comment]
```
07-10 23:42:10.149  8117  8117 E AndroidRuntime: FATAL EXCEPTION: main
07-10 23:42:10.149  8117  8117 E AndroidRuntime: Process: app.revanced.android.youtube, PID: 8117
07-10 23:42:10.149  8117  8117 E AndroidRuntime: java.lang.VerifyError: Verifier rejected class aaqi: void aaqi.Q(boolean): [0xFFFFFFFF] invalid arg count (0) in non-range invoke (declaration of 'aaqi' appears in /data/app/~~RemO-xizxTgI-nZTmN9vfw==/app.revanced.android.youtube-oq4aduX2T-9bzT4lVT7xXw==/base.apk!classes4.dex)
```

**Comment by TheJeterLP** [comment]
Yeah so the fingerprint probably broke, I'll have a look at it tomorrow, or you can fix it if you want

**Comment by TheJeterLP** [comment]
fixed with #148
